ctriface/iface.go

```diff
@@ -33,6 +32,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/vhive-serverless/vhive/snapshotting"
```

Comment:
this doesn't look correct to me. memory manager should be the one that includes this module

Reply:
this module is imported because `snap *snapshotting.Snapshot` is used in several function parameters in iface.go.